### PR TITLE
Further improve README

### PR DESCRIPTION
**Pull request by raffaem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,15 @@ LibreOffice SDK is not required.
 
 To build:
 
-1.  Run `scripts/symlink_sdk`
+1.  Run `scripts/symlink_sdk`.
 1.  Open Eclipse and import this project into your workspace.
-1.  Right-click the project in the Eclipse Package Explorer, click "Properties", and then select "Java Build Path." Click the libraries tab and ensure that all referenced files exist, or else correct the paths.
+1.  Right-click the project in the Eclipse Package Explorer, click "Properties", select "Java Build Path", then select the "Libraries" tab. Click on "Add JARs" and expand the "Zotero OpenOffice Integration" folder. Make sure there is no "lib" subfolder underneath it. If there is:
+	1. Expand "Zotero OpenOffice Integration" -> "lib" -> "libreoffice-sdk" and select all the JARs underneath it.
+	1. Click "Ok" and then "Apply and Close".
```
Comment (adomasven) [inline README.md:19]: This is not something that needs to be done. The plugin compiles fine without this step. Could you expand why you want to add this?

**Comment by raffaem** [comment]
Sure.

As it is now, Eclipse will show errors on the imports:
image
If you point your mouse in the lightbulb, it will show a message "The import com.sun.star.awt cannot be resolved".

It will also show the same errors in the code:
image

It will also shows error at compile time:
image

The file will compile nonetheless, but when you install it, LibreOffice will show "NoClassDefFound" error and won't install it:
image

But maybe I'm doing something wrong at a more fundamental level.

I have little experience with Java and no experience with LibreOffice plugins.

**Comment by adomasven** [comment]
I see this sort of stuff if I have OpenJDK selected in Eclipse as my JRE instead of the Oracle JDK. Which JRE are you using in Eclipse? Does doing this actually fix it? If we merge #70, is this even necessary?

**Comment by raffaem** [comment]
I have Oracle JDK:

image

I don't have openjdk even installed:
```
sudo dnf list --installed | grep openjdk
```

> Edit: Also when compiling without these libraries with Oracle JDK, the plugin installs fine into LibreOffice.

It doesn't for me. It shows the error I reported above.

> If we merge https://github.com/zotero/zotero-libreoffice-integration/pull/70, is this even necessary?

Maybe the JARs will change over time, so I thought the user should be aware on how to add them if something changes.

**Comment by adomasven** [comment]
Sorry, I am still confused. How did you develop the other PRs prior to this one? It seemed like you tested your own changes yourself. Were you already doing this?

> In any case, we should have Eclipse show only the real errors, otherwise you can't tell which errors are harmless and which ones are real errors.

Eclipse does not show any errors for me without these libs. But either way, those are not just cosmetic, if you these errors, then the compiled JAR won't work.

Does adding these libs actually fix the errors for you? Are you sure it wasn't something else in Eclipse? I don't often work with Eclipse, but it's quite finicky and e.g. if I don't have the sdk linked before importing the project into eclipse, or have the wrong JRE selected, it doesn't fix itself until i remove the project from Eclipse and reimport it.

**Comment by raffaem** [comment]
> Sorry, I am still confused. How did you develop the other PRs prior to this one? It seemed like you tested your own changes yourself. Were you already doing this?

Yes, but I git added only the relevant changes.

> Does adding these libs actually fix the errors for you?

Yes

> Are you sure it wasn't something else in Eclipse?

No I'm not. I don't regularly use Eclipse, I used it A LOT of years ago and now again only for this extension.

> Eclipse does not show any errors for me without these libs

Are you using the `build/.classpath` in the repository?

What version of Oracle JDK do you have?

What version of LibreOffice?

What OS?

**Comment by raffaem** [comment]
So Let's go step by step on this.

It looks like `com.sun.star.awt.MessageBoxButtons` (imported by `Document.java`) is defined in `libreoffice.jar`.

There is no `libreoffice.jar` in `build/.classpath`.

How is your Eclipse resolving that class?

**Comment by adomasven** [comment]
Huh, well, I don't even know what to tell you here. I'm on Ubuntu 20.04, LibreOffice 7.2, Oracle JDK 18. I've been building this repository for years with the current .classpath and it was fine. I remember there always being more files in the libreoffice libs file, but I never got errors in Eclipse. Yesterday when I spun up Eclipse again for this, I did have to reimport the project a few times, but I assumed it's because I didn't first link libreoffice sdks, and then didn't have Oracle JDK installed and selected, and Eclipse is just finicky. Today I'm opening the project and I'm getting these errors again. Adding the remaining .jar files to the build path from libreoffice sdk fixes them.

And yeah, I'm looking at the definitions from the jars and yes, the classes the plugin used are indeed defined in libreoffice.jar and others. This is an old repository that I myself inherited from the original dev, so perhaps at some point LibreOffice libs got split into more files, but I couldn't say when.

Either way, thanks for walking through this with me. I'll merge this along with #70

**Pull request merged**